doc/api/errors.md

```diff
@@ -450,18 +452,34 @@ added properties.
 
 #### error.code
 
-Returns a string representing the error code, which is always `E` followed by
-a sequence of capital letters, and may be referenced in `man 2 intro`.
+The `error.code` property is a string representing the error code, which is always
+`E` followed by a sequence of capital letters.
 
 #### error.errno
 
-Returns a number corresponding to the **negated** error code, which may be
-referenced in `man 2 intro`. For example, an `ENOENT` error has an `errno` of
-`-2` because the error code for `ENOENT` is `2`.
+The `error.errno` property is a number or a string.
+The number corresponds to the **negated** error code. For example, an `ENOENT`
+error has an `errno` of `-2` because the error code for `ENOENT` is `2`.
+In case of a string, it is the same as `error.code`.
 
 #### error.syscall
 
-Returns a string describing the [syscall][] that failed.
+The `error.syscall` property is a string describing the [syscall][] that failed.
+
+#### error.path
+
+When present (e.g. in `fs` or `child_process`), the `error.path` property is a string
+containing a relevant invalid pathname.
+
+#### error.address
+
+When present (e.g. in `net` or `dgram`), the `error.address` property is a string
+describing the address to which the connection failed.
+
+#### error.port
+
+When present (e.g. in `net` or `dgram`), the `error.port` property is a number of
+the connection's port that is not available.
```

Comment:
"port that is not available" sounds too specific to me. Is port binding the only case in Node.js when `err.port` is present? Either way, I'd prefer "a relevant port number", which is intentionally vague but more future-proof.

Also, "... property is a number representing ..." sounds slightly better.